docs/concepts/resource-limits-and-performance-optimization.md

```diff
@@ -99,8 +108,46 @@ While the resources limits on CPU and memory usage, crash tolerance, UI responsi
 
 - Test your add-in against the highest volume of data you expect, and restrict your add-in to process up to that limit.
 
-- Minimize data exchanges between the add-in and the Office document. For more information, see [Avoid using the context.sync method in loops](correlated-objects-pattern.md).
+### Performance improvements with the host-specific APIs
+
+The performance tips in [Using the host-specific API model](../develop/host-specific-api-model.md) provide guidance when using the host-specific APIs for Excel, OneNote, Visio, and Word. In summary, you should:
+
+- [Only load necessary properties](../develop/host-specific-api-model.md#calling-load-without-parameters-not-recommended).
+- [Minimize the number of sync() calls](../develop/host-specific-api-model.md#performance-tip-minimize-the-number-of-sync-calls). Read [Avoid using the context.sync method in loops](correlated-objects-pattern.md) for further information on how to avoid `sync` calls in your code.
+- [Minimize the number of proxy objects created](../develop/host-specific-api-model.md#performance-tip-minimize-the-number-of-proxy-objects-created). You can also untrack proxy objects, as described below.
```

Comment:
Usually avoid terms like "above" and "below" since that relative location may not be the same in certain languages.
```suggestion
- [Minimize the number of proxy objects created](../develop/host-specific-api-model.md#performance-tip-minimize-the-number-of-proxy-objects-created). You can also untrack proxy objects, as described in the next section.
```